torch/lib/c10d/ProcessGroupNCCL.hpp

```diff
@@ -197,9 +198,7 @@ class ProcessGroupNCCL : public ProcessGroup {
   //
   // If created by WorkNCCL's getFuture API, FutureNCCL has a reference to
   // WorkNCCL's cudaEvents, NCCL collective's outputs, and the device index of
```

Comment:
index -> indices?

Reply:
Thanks, fixed (it'll be visible once I re-export it)

I was thinking of doing a complete pass over comments/docstrings/... at the very end, once we have split out CUDAFuture, to document the reasons behind our design choices, the subtleties around streams usage, and such.